pkg/scheduler/framework/plugins/podtopologyspread/scoring_test.go

```diff
@@ -736,6 +736,35 @@ func TestPodTopologySpreadScore(t *testing.T) {
 				{Name: "node-b", Score: 0},
 			},
 		},
+		{
+			// Actually, you cannot create Pods which have both non-default MinDomains and SchedulingAnyway
+			// through kube-apiserver.
```

Comment:
```
// This test is artificial. In the real world, API Server would fail a pod's creation
// when non-default minDomains is specified along with SchedulingAnyway.
```